src/algos/topologies/base.py

```diff
@@ -25,7 +26,15 @@ def generate_graph(self) -> None:
         NetworkX has a lot of built-in functions to generate graphs
         Use this url - https://networkx.org/documentation/stable/reference/generators.html
         """
-        pass
+
+        # The graph was never initialized check
+        assert(self.graph is not None)
+
+        # Draw the graph
+        nx.draw(self.graph, with_labels=True, node_color="lightblue", font_weight="bold")
+
+        # Display the graph
+        plt.show()
```

Comment:
What's the purpose of this?

Reply:
I am an idiot no good reason. Removing from PR.